### PR TITLE
🎉 New method: prepend (132 B)

### PR DESCRIPTION
**Pull request by BabkinAleksandr**
[Ramda prepend](http://ramdajs.com/docs/#prepend)

aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils

### CODE REVIEW

File: lib/prepend/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function prepend(elem, list) {
+  return [elem].concat(list)
+})
```
Comment (Beraliv) [inline lib/prepend/index.js]: use `var`, `for(;;)` and speed it up, please
or at least check it would be faster

Comment (BabkinAleksandr) [reply lib/prepend/index.js]: `concat` is much faster than `for`:

`for`:
aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils

index_js_ _prepend_ ___documents_nanoutils

Comment (Beraliv) [reply lib/prepend/index.js]: Guys, great news!
We can improve performance using `Array` constructor and setting values with `arr[i] = list[i]`
it's even faster than `concat`

**Comment by Beraliv** [comment]
image

`ramda` is so slow after all, ahahah

**Comment by Kelin2025** [comment]
YES, SIR!

**Pull request merged**